src/rgw/rgw-restore-bucket-index

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
+
+# version 2023-03-03
```

Comment:
as a repair tool, i think this deserves a concise description of what it does and when you'd want to use it